### PR TITLE
Setup dependabot with weekly pnpm and GA updates

### PR DESCRIPTION
**Pull request by jasikpark**
With weekly runs, this should keep nebula-docs up-to-date with minimal interaction and automatic PR creation.

Modelling the slackhq/nebula file: https://github.com/slackhq/nebula/blob/master/.github/dependabot.yml

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    schedule:
+      interval: 'weekly'
+    directories:
+      - '/'
+    groups:
+      docusaurus-dependencies:
```
Comment (jasikpark) [inline .github/dependabot.yml:9]: these should group our dependabot and react deps into their own prs

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
```
Comment (IanVS) [inline .github/dependabot.yml:3]: Should this be `pnpm`?  https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#pnpm

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    schedule:
+      interval: 'weekly'
```
Comment (IanVS) [inline .github/dependabot.yml:5]: I'd prefer monthly, but if you want to try this for a bit, that's fine.

**Review by IanVS — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (jasikpark) [reply .github/dependabot.yml:3]: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

clarifies that it's supported via the `npm` yaml value

**Pull request merged**